app/controllers/admin/screened_ip_addresses_controller.rb

```diff
@@ -44,11 +46,6 @@ def destroy
     render json: success_json
   end
 
-  def roll_up
```

Comment:
Usually I would suggest that we need to keep backwards compatibility here for people that might be depending on this using the API. However, I don't think it is worth the effort here given what the endpoint is doing. We're working on a public/private endpoints split so this should improve in the future.